src/components/Details/carsDetails.jsx

```diff
@@ -0,0 +1,98 @@
+import React, { useEffect, useState } from 'react';
+import { useSelector } from 'react-redux';
+import { useAuthHeader } from 'react-auth-kit';
+import { useParams, useDispatch } from 'react-router-dom';
```

Comment:
`useDispatch` comes from react-redux not react-router-dom. Please move the import of useDispatch to come from the correct library